files/scripts/supervisor-proc-exit-listener

```diff
@@ -86,8 +101,16 @@ def generate_alerting_message(process_name, status, dead_minutes):
     else:
         namespace = namespace_prefix + namespace_id
 
-    syslog.syslog(syslog.LOG_ERR, "Process '{}' is {} in namespace '{}' ({} minutes)."
-                  .format(process_name, status, namespace, dead_minutes))
+    message = "Process '{}' is {} in namespace '{}' ({} minutes).".format(
+                                                                    process_name,
+                                                                    status,
+                                                                    namespace,
+                                                                    dead_minutes)
+    if is_warm_reboot():
```

Comment:
>is_warm_reboot [](http://example.com/codeflow?start=7&length=14)

Could this condition be more conservative? like is_warm_reboot and the orchagent is stuck at the same time?

Reply:
Fixed, only ignore error log for orchagent stuck during warm reboot.

Reply:
Seems like `WARM_RESTART_ENABLE_TABLE` is the indicator for warm-boot-up, not warm-shutdown which is what you want. I am considering orchagent process has the full visibility of freezing status, how about continuing hearbeat even when it is frozen?

Sorry for misleading.

Reply:
Currently orchagent freeze by sleep, I will check if can improve the restart code in orchagent by stop handle all task:

                    SWSS_LOG_WARN("Orchagent is frozen for warm restart!");
                    sleep(UINT_MAX);